docs/Hooks.md

```diff
@@ -293,6 +294,37 @@ fastify.addHook('onRoute', (routeOptions) => {
   routeOptions.prefix
 })
 ```
+<a name="on-register"></a>
+**'onRegister'**<br>
+Triggered when a new plugin function is registered, and a new encapsulation context is created, he hook will be executed **before** the plugin code.<br/>
```

Comment:
spelling `he` -> `the`

Reply:
Can you use the "suggest" feature of GitHub? :)
https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request